prototype/Frontend/user_replies.html

```diff
@@ -84,13 +59,15 @@
     <link href="/public/css/helpercss/swiper-bundle.min.css" rel="stylesheet" />
 
     <style>
-      .btn-primary {
-        padding: 0 !important;
-        width: 6rem !important;
-        height: 2.2rem !important;
-        background-color: #2b83f3;
-      }
-    </style>
+        .btn-primary {
+          padding: 0 !important;
+          width: 6rem !important;
+          height: 2.2rem !important;
+          background-color: #2b83f3;
+        }
+
+     </style>
+
 
     <!-- charts -->
     <script
```

Comment:
The scripts are supposed to be at the end of the body tag .. please dont add to the header